calico/getting-started/kubernetes/self-managed-public-cloud/aws.md

```diff
@@ -22,8 +22,10 @@ Kubernetes Operations (kops) is a cluster management tool that handles provision
 
 
 > **Note**: {{site.prodname}} makes use of the Kubernetes Container Storage Interface (CSI) to support various types of volumes. The necessary drivers required for CSI
-> to function correctly in EKS clusters will no longer be present by default in clusters running Kubernetes 1.23. Please see the following link to ensure your cluster
+> to function correctly in AWS clusters using EBS volumes will no longer be present by default in clusters running Kubernetes 1.23. Please see the following link to ensure your cluster
```

Comment:
I think this is going to depend on what installer you're using.  I don't think you can say that _all_ methods of installing kubernetes clusters 1.23+ on AWS will not have EBS CSI by default?

Reply:
True it will largely depend on what installer you're using. I'll reword it to indicate as much.